hugolib/author_test.go

```diff
@@ -0,0 +1,408 @@
+package hugolib
+
+import (
+	"reflect"
+	"testing"
+)
+
+var (
+	defaultAuthorsMap = map[string]interface{}{
+		"derek": map[string]interface{}{
+			"givenName":   "Derek",
+			"familyName":  "Perkins",
+			"displayName": "Derek Perkins",
```

Comment:
Remove this `displayName` to check that the default `DisplayName` construction works.